ledger/partial/ptrie/partialTrie.go

```diff
@@ -24,7 +24,7 @@ type PSMT struct {
 
 // RootHash returns the rootNode hash value of the SMT
 func (p *PSMT) RootHash() ledger.RootHash {
-	return ledger.RootHash(p.root.HashValue())
+	return ledger.RootHash(p.root.Hash())
```

Comment:
Just a sanity check: is it possible for an empty partial trie to have a `nil` root? 
I think no because I see a [new node](https://github.com/onflow/flow-go/blob/alex/5346-mtrie-array-hashes_-_suggestions_pt2/ledger/partial/ptrie/partialTrie.go#L80 ) created in `NewPSMT`

Reply:
Good catch. I feel for consistency, a `PSMT` should be structured as similar as possible, just with some branches pruned. Having a root node even for the empty trie is probably a legacy from the previous version of the complete ledger trie. I am ok leaving this small tech debt for now, but if you are motivated to clean it up, I would be very happy.

Reply:
I'll work on the consistency as part of https://github.com/dapperlabs/flow-internal/issues/1488